commands/command.go

```diff
@@ -27,19 +28,25 @@ func (cmd *BaseCommand) Before(c *cli.Context) error {
 	util.LoggerInit(c.GlobalBool("verbose"))
 	cmd.out = util.Logger()
 	cmd.machine = Machine{Name: c.GlobalString("name"), out: util.Logger()}
+
+	util.NotifyInit(fmt.Sprintf("Outrigger (rig) %s", c.App.Version))
+	cmd.context = c
```

Comment:
Why do we need to stash context? This is begging for a why comment.

Reply:
Ah, we need it so Error() and Success() do not need ctx passed in. Could still use a comment to point that out.

Reply:
ok